diff --git a/node_modules/react-native-slider/lib/Slider.js b/node_modules/react-native-slider/lib/Slider.js
index c640410..1eef749 100644
--- a/node_modules/react-native-slider/lib/Slider.js
+++ b/node_modules/react-native-slider/lib/Slider.js
@@ -6,3 +6,3 @@ var _react=require("react");var _react2=_interopRequireDefault(_react);
 
-var _reactNative=require("react-native");
+var _reactNative=require("react-native");var _deprecatedReactNativePropTypes=require("deprecated-react-native-prop-types");
 
@@ -551,9 +551,9 @@ return false;}}]);return Slider;}(_react.PureComponent);Slider.propTypes={ /**
      * The style applied to the slider container.
-     */style:_reactNative.ViewPropTypes.style, /**
+     */style:_deprecatedReactNativePropTypes.ViewPropTypes.style, /**
      * The style applied to the track.
-     */trackStyle:_reactNative.ViewPropTypes.style, /**
+     */trackStyle:_deprecatedReactNativePropTypes.ViewPropTypes.style, /**
      * The style applied to the thumb.
-     */thumbStyle:_reactNative.ViewPropTypes.style, /**
+     */thumbStyle:_deprecatedReactNativePropTypes.ViewPropTypes.style, /**
      * Sets an image for the thumb.
-     */thumbImage:_reactNative.Image.propTypes.source, /**
+     */thumbImage:_deprecatedReactNativePropTypes.ImagePropTypes.source, /**
      * Set this to true to visually see the thumb touch rect in green.
diff --git a/node_modules/react-native-slider/src/Slider.js b/node_modules/react-native-slider/src/Slider.js
index 37deee5..03eb613 100644
--- a/node_modules/react-native-slider/src/Slider.js
+++ b/node_modules/react-native-slider/src/Slider.js
@@ -1 +1,2 @@
+import { ViewPropTypes, ImagePropTypes } from 'deprecated-react-native-prop-types';
 'use strict';
@@ -148,3 +149,3 @@ export default class Slider extends PureComponent {
      */
-    thumbImage: Image.propTypes.source,
+    thumbImage: ImagePropTypes.source,
 
