diff --git a/node_modules/react-native-action-button/ActionButton.js b/node_modules/react-native-action-button/ActionButton.js
index 5ee6888..13b3bd0 100644
--- a/node_modules/react-native-action-button/ActionButton.js
+++ b/node_modules/react-native-action-button/ActionButton.js
@@ -1 +1,2 @@
+import { TextPropTypes } from 'deprecated-react-native-prop-types';
 import React, { Component, useState, useRef, useEffect } from "react";
@@ -338,3 +339,3 @@ ActionButton.propTypes = {
   buttonColor: PropTypes.string,
-  buttonTextStyle: Text.propTypes.style,
+  buttonTextStyle: TextPropTypes.style,
   buttonText: PropTypes.string,
